### DISCUSSION TITLE
Upgrade path for Frigate

### DISCUSSION THREAD

**Discussion by modem158** [tteck/Proxmox#3671] [Q&A]
I know this has been asked a lot as I have searched for weeks for an answer that works.  I've found alot of information but nothing that has led me to a path forward.  Please feel free to just link me to instructions that you know of...

I have Frigate 0.14.0 setup in Proxmox using the TTECK scripts.  I would like to grab the 0.14.1 bug fixes and improvements but I can't find a clear path to upgrade.  I know I can just blow away the LXC and create a new one using the script but I would then have to re-link my coral USB passthrough and go through that setup again.  Is there any way for me to just point the existing container at the new build and have it pull in 0.14.1 and all of its dependencies?

**Comment by danbutter** [comment]
Sorry I can't help.  I was in the same situation yesterday and saw this:
https://github.com/tteck/Proxmox/discussions/3627
I just did the make a new one too.  Not the most convenient, but still better than installing frigate manually.  I bet if he comes up with a way to do the update in the future he'll let us know.

Maybe someone else who knows how this is setup can come up with a way.
